integration-tests/src/test/resources/wdt-models/coherence-managed-wdt-config-mii.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+domainInfo:
+    AdminUserName: '@@PROP:AdminUserName@@'
+    AdminPassword: '@@PROP:AdminPassword@@'
+    ServerStartMode: 'prod'
+topology:
+    Name: 'miidomain'
+    AdminServerName: 'admin-server'
+    Cluster:
+        'cluster-1':
+            CoherenceClusterSystemResource: CoherenceCluster
+            CoherenceTier:
+                CoherenceWebFederatedStorageEnabled: false
+                CoherenceWebLocalStorageEnabled:     false
+                LocalStorageEnabled:                 false
+            DynamicServers:
+                ServerNamePrefix: 'cluster-1-managed-server'
+                MaxDynamicClusterSize: 5
+                CalculatedListenPorts: false
+                MaximumDynamicServerCount: 5
+                DynamicClusterSize: 5
+                ServerTemplate: 'cluster-1-template'
+        'cluster-2':
+            CoherenceClusterSystemResource: CoherenceCluster
+            CoherenceTier:
+                CoherenceWebFederatedStorageEnabled: false
+                CoherenceWebLocalStorageEnabled:     false
+                LocalStorageEnabled:                 true
+            DynamicServers:
+                ServerNamePrefix: 'cluster-2-managed-server'
+                MaxDynamicClusterSize: 5
+                CalculatedListenPorts: false
+                MaximumDynamicServerCount: 5
+                DynamicClusterSize: 5
+                ServerTemplate: 'cluster-2-template'
+    Server:
+        'admin-server':
+            ListenPort: 7001
+    ServerTemplate:
+        'cluster-1-template':
+            ListenPort: 8001
+            Cluster: 'cluster-1'
+            CoherenceClusterSystemResource: CoherenceCluster
+            CoherenceMemberConfig:
+              UnicastListenAddress:  'miidomain-cluster-1-managed-server${id}'
+        'cluster-2-template':
+            ListenPort: 8001
+            Cluster: 'cluster-2'
+            CoherenceClusterSystemResource: CoherenceCluster
+            CoherenceMemberConfig:
```

Comment:
This will not be needed after my PR #2499 .  Please remove it after PR #2499 is merged.